### ISSUE TITLE
Add role = "navigation" to mainNavbar

### ISSUE THREAD

**Issue by brylie** [refugeetech/platform#51]
The mainNavbar nav currently lacks accessibility feature, called role. Add `role = "navigation"` to the `nav` element in the `mainNavbar` template.

**Comment by Angamanga** [comment]
I can fix this one too!

**Comment by Angamanga** [comment]
Please check pull request from branch feature/issue51
